aiohttp/client_reqrep.py

```diff
@@ -230,6 +243,8 @@ def __init__(
         method: str,
         url: URL,
         *,
+        response_params: _ResponseParams,
+        timeout: ClientTimeout,
         params: Query = None,
         headers: Optional[LooseHeaders] = None,
         skip_auto_headers: Optional[Iterable[str]] = None,
```

Comment:
I'm just thinking through future maintainability of this class. Should we put in a `**kwargs: Any` and not use it, so that any subclass is required to include that and pass it through the super() call? That way we should be pretty safe to add parameters in future without worrying about breaking any subclasses.